force-app/main/default/lwc/edaSettings/edaSettings.html

```diff
@@ -1,11 +1,11 @@
 <template>
     <template if:true={currentUserHasAccess}>
         <div class="slds-grid slds-theme_default eda-height_full">
-            <div class="slds-col slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full">
-                {labelReference.settingsNavigation}
+            <div class="slds-col slds-p-top_large slds-no-flex slds-size_4-of-12 slds-scrollable_y eda-height_full">
```

Comment:
the paddling looks alright but should we consider having a consistent way to have paddings and margins aligned? Also, I am curious about eda-height_full. Do we really need a custom css?

Reply:
For the padding, that's something we'll want to look at once we have a 'full' UI and establish where the padding does and doesn't go relative to what we have displayed. My understanding is that's usually done at a 'fit and finish' stage once we have the components together.

Unfortunately, we do need the 100% height for the time being, though if you can find an slds class that is supported and documented that handles the height, that would be the superior solution.